src/sgame/sg_main.cpp

```diff
@@ -2273,6 +2276,10 @@ can see the last frag.
 */
 void CheckExitRules()
 {
```

Comment:
Use Cvar::Cvar<bool> g_neverEnd(<stuff>); so you don't have to put it in the array nor declare it in sg_extern.h